src/blurredLocation.js

```diff
@@ -81,6 +84,8 @@ BlurredLocation = function BlurredLocation(options) {
     });
   };
 
+  panMapToGeocodedLocation(options.selector);
```

Comment:
do you need to run this here? it should only be needed in the example, right? no need to run it before someone's geocoded anything, i think.